include/swift/Reflection/RuntimeInternals.h

```diff
@@ -108,11 +108,12 @@ struct ActiveTaskStatusWithoutEscalation {
 
 template <typename Runtime, typename ActiveTaskStatus>
 struct AsyncTaskPrivateStorage {
+  typename Runtime::StoredPointer ExclusivityAccessSet[2];
   ActiveTaskStatus Status;
   StackAllocator<Runtime> Allocator;
   typename Runtime::StoredPointer Local;
-  typename Runtime::StoredPointer ExclusivityAccessSet[2];
   uint32_t Id;
+  uint32_t BasePriority;
```

Comment:
@mikeash, not sure if you are surfacing up this information but I added this to the AsyncTaskPrivateStorage as part of https://github.com/apple/swift/pull/40718 so am modifying it here to reflect that.

Reply:
Sounds good. I think we'll want to surface it even if we don't yet do so. Once this goes in I'll take a pass through swift-inspect and make sure it's showing all relevant new info.